kernel/classes/ezcontentobjecttreenodeoperations.php

```diff
@@ -113,6 +113,7 @@ static function move( $nodeID, $newParentNodeID )
                 $nodeIDList = array( $nodeID );
                 eZSearch::removeNodeAssignment( $node->attribute( 'main_node_id' ), $newNode->attribute( 'main_node_id' ), $object->attribute( 'id' ), $nodeIDList );
                 eZSearch::addNodeAssignment( $newNode->attribute( 'main_node_id' ), $object->attribute( 'id' ), $nodeIDList, true );
+                eZSearch::refreshChildren( $node );
```

Comment:
Using `addNodeAssignment` when we really want to reindex a subtree feels wrong to me. I agree with @peterkeung and @pkamps here. Require cronjob to be run, that is, act as if delayed indexing is turned on.

I have no problems with adding a case for moved nodes to `ezplatformsearch` and documenting it.